source/common/stats/symbol_table_impl.h

```diff
@@ -320,11 +375,28 @@ class StatName {
     return symbol_array_[0] | (static_cast<uint64_t>(symbol_array_[1]) << 8);
   }
 
+  const uint8_t* symbolArray() const { return symbol_array_; }
+
+  /**
+   * @return uint64_t the number of bytes in the symbol array, including the two-byte
+   *                  overhead for the size itself.
+   */
+  uint64_t numBytesIncludingLength() const { return numBytes() + StatNameSizeEncodingBytes; }
```

Comment:
The difference between numBytes() and numBytesIncludingLength() has become sort of subtle. WDYT of renaming these to arraySize() and size() ? That way it's clear that the whole object has a certain size(), or you can request the size of some subcomponent.

Reply:
Good idea; naming is hard for me sometimes. Helps to see another perspective. I changed:

numBytesIncludingLength() -> size()
numBytes() -> dataSize()
symbol_array_ -> size_and_data_

and I removed the accessor to symbolArray().